wp-content/themes/core/blocks/core/lists/style.pcss

```diff
@@ -0,0 +1,47 @@
+/**
+ * Styles specific to this block
+ */
+
+/* ----------------------------------------------------------------------
+ * Ordered List
+ * ---------------------------------------------------------------------- */
+
+:where(.wp-block-post-content) ol,
```

Comment:
This is a nice way to handle this. I love it. I wonder if we should document somewhere all the different WP-Editor css classes that are available and where/why we're using them?